config/defaultMappingTemplate.json

```diff
@@ -138,12 +123,10 @@
                 "type" : "boolean"
```

Comment:
We should probably talk about whether we want doc_values here or keep it as is. The question is would we ever want to sort/aggregate on a boolean field.